index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <nav class="navbar">
+      <div class="log">My Logo</div>
+      <a href="#" class="header-button">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </a>
+      <div class="nav-links">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="background-img">
+
+      <div class="row">
+        <div class="main">
+          <div>I'm Naol</div>
+          <div>Glad to see</div>
+          you!
+        </div>
+        <div class="description">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
```

Comment:
- [ ] It would be semantically correct to make this section a paragraph instead of a div element.
- [ ] Also, kindly ensure that you make use of the provided font size for this paragraph. This way you will have stuck to the project design and met all the project requirements.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/191768113-9aedc38c-5475-4423-ac58-8aaadffdc963.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/191768243-34337587-9daa-4b10-b76f-c25fbd0b4f15.png)